projects/ion/src/lib/checkbox/checkbox.component.scss

```diff
@@ -0,0 +1,121 @@
+@import '../../styles/colors/colors.scss';
+
+@mixin setCheck($color) {
+  content: url("data:image/svg+xml,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='%23000' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath id='Union' d='M10.8839 2.28075C11.3721 1.79258 11.3721 1.00113 10.8839 0.512978C10.3957 0.0248294 9.60428 0.0248399 9.11613 0.513002L4.7939 4.83535L2.88385 2.9253C2.3957 2.43714 1.60424 2.43714 1.11609 2.9253C0.627931 3.41345 0.627931 4.20491 1.11609 4.69307L3.91003 7.48701C4.14445 7.72143 4.4624 7.85313 4.79392 7.85313C5.12545 7.85312 5.44339 7.72142 5.67781 7.487L10.8839 2.28075Z' fill='%23" + $color + "'/%3E%3C/svg%3E");
+}
+
+@mixin setDash($color) {
+  content: url("data:image/svg+xml,%3Csvg width='24' height='24' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M0 1C0 0.447715 0.447715 0 1 0H9C9.55228 0 10 0.447715 10 1V3C10 3.55228 9.55228 4 9 4H1C0.447716 4 0 3.55228 0 3V1Z' fill='%23" + $color + "'/%3E%3C/svg%3E%0A");
+}
+
+$color-primary-2: 'b5d2fc';
+$color-primary-1: 'ebf3fe';
+$color-neutral-4: 'ced2db';
```

Comment:
These colors are already defined in index.scss, you don't need to declare them, just use the colors like $primary-1 or $primary-color, etc.

Reply:
turns out I needed the variables without the '#' as they are defined in colors.css. Because of this i create the local variables referencing the expecific color